app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -47,6 +47,8 @@ def translate_exception(err)
       MiqException::MiqHostError.new "Socket error: #{err.message}"
     when MiqException::MiqInvalidCredentialsError, MiqException::MiqHostError
       err
+    when Net::HTTPBadResponse
+      Net::HTTPBadResponse.new "Login failed due to a bad hostname and/or port and/or security protocol."
```

Comment:
May I suggest a simple `Login failed due to bad security protocol, hostname or port.`?